wagtail/admin/views/reports/page_types_usage.py

```diff
@@ -39,7 +39,7 @@ def _annotate_last_edit_info(queryset, language_code, site_root_path):
 
     latest_edited_page = Page.objects.filter(
         content_type=OuterRef("pk"), **latest_edited_page_filter_kwargs
-    ).order_by("-latest_revision_created_at", "title", "-pk")
+    ).order_by(F("latest_revision_created_at").desc(nulls_last=True), "title", "-pk")
```

Comment:
🤌

Reply:
It took me so much time to figure out why some tests were failing only on some DB backends :laughing:

Reply:
I've definitely been there :)